hedyorg/hedy
# 🪲Fix not being able to complete the quiz while in programmer's mode

If the developers mode was enabled, then the parsons and the quiz became unusable. To solve this I split the logic into two functions: one to set the developers mode and another to show it. Now you can actually complete the quiz and the parsons while in this mode.

Fixes #5172 

**How to test**

* Check the programmer's mode toggle while being logged off and check that you can complete the quiz and the parsons.
* As a student of a class with mandatory programmer's mode check that you can complete the quiz and the parsons (I wonder if teachers use this feature much being that no one complained about this before recently)
